public/API/API_GetConsoleIDs.php

```diff
@@ -9,6 +9,20 @@
  *   object    [value]
  *    string    ID                  unique identifier of the console
  *    string    Name                name of the console
+ *    string    IconURL             System icon URL
  */
 
-return response()->json(System::select(['ID', 'Name'])->get());
+$systems = System::select()->get();
+
+$response = [];
+
+foreach ($systems as $system) {
+    $data = [
+        'ID' => $system['ID'],
+        'Name' => $system['Name'],
+        'IconURL' => $system->icon_url,
+    ];
+    $response[] = $data;
+}
+
+return response()->json($response);
```

Comment:
A `map()` operator is a bit more concise:

```php
$systems = System::all()->map(function ($system) {
    return [
        'ID' => $system->ID,
        'Name' => $system->Name,
        'IconURL' => $system->icon_url,
    ];
});

return response()->json($systems);
```

Reply:
Updated with this change. 
Thanks for the suggestion, PHP is not my jam 😅